ios/RealmWrapper.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  RealmWrapper.swift
+//  COVIDSafePaths
+//
+//  Created by Tyler Roach on 4/23/20.
+//  Copyright © 2020 Path Check Inc. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+class RealmWrapper: NSObject {
+  
+  @objc static let shared = RealmWrapper()
+  
+  private static let MINIMUM_TIME_INTERVAL = 60 * 4
+  private static let DAYS_TO_KEEP = 14
+  private static let KEY_LAST_SAVED_TIME = "lastSavedTime"
+  
+  let realmConfig: Realm.Configuration
+  
+  private override init() {
+    realmConfig = Realm.Configuration(objectTypes: [Location.self])
```

Comment:
we should call super.init() here

Reply:
Pushing up encryption work shortly which will remove this override.